cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -201,11 +201,17 @@ script "js_ri_types", type: "javascript" do
   result "result"
   code <<-EOS
     var result = []
+    var services = ["ElastiCache","Elastic Compute Cloud (EC2)","Elasticsearch Service (ES)","Redshift","Relational Database Service (RDS)"]
     if (param_service === "All"){
-      var services = ["ElastiCache","Elastic Compute Cloud (EC2)","Elasticsearch Service (ES)","Redshift","Relational Database Service (RDS)"]
       _.each(services, function(service){
         result.push({"service":service})
       })
+    } else if (param_service === "All Except EC2"){
+      services.forEach(function(service) {
+        if (service !== "Elastic Compute Cloud (EC2)") {
+          result.push({"service":service})
+        }
+    });
```

Comment:
Seems like it would be better to add a new param (`param_services_exclude`) instead so it's more extendable (?)

Reply:
good thought!, As per ticket, "add new selection to service input"

https://flexera.atlassian.net/browse/HCLIND-1

I will leave it to @cdwilhelm

Reply:
@degarajesh    The goal is to exclude services included in the Savings Plans so the savings are not duplicated.  There are many ways to solve this.  
I can see more services covered by Savings Plans in the future, such as Databases. Bryan raises a good point to make it extensible.  
@bryankaraffa   how do you see a user of this policy configuring it to exclude EC2 or other services?  What options  would they select in the param_services and param_services_excluded